kube_log_watcher/agents/__init__.py

```diff
@@ -2,10 +2,12 @@
 
 from kube_log_watcher.agents.appdynamics import AppDynamicsAgent
 from kube_log_watcher.agents.scalyr import ScalyrAgent
+from kube_log_watcher.agents.symlinker import SymlinkerLoader
 
 
 __all__ = (
     AppDynamicsAgent,
     BaseWatcher,
     ScalyrAgent,
+    SymlinkerLoader
```

Comment:
Just styling: adding `,` at the end of the last element in the dict 😉

Reply:
👍  I will merge this and add the comma with the next PR (which will bump the version).